source/pug/partials/events.pug

```diff
@@ -5,29 +5,29 @@
 
 .container
   include:markdown ../../markdown/events.md
-  .events
-    .event-container
+  .events.row
+    .event-container.col-md-3.col-sm-6.col-6
```

Comment:
https://v4-alpha.getbootstrap.com/layout/grid/#grid-options

Bootstrap is based on 12-column grid. So `col-md-3` means "make this element 3-column wide on medium size devices"